src/core/Checkbox/Checkbox.tsx

```diff
@@ -111,12 +119,36 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
           style={checkmarkStyle}
         >
           {isLoading && <ProgressRadial indeterminate />}
-          {!isLoading && (
-            <svg viewBox='0 0 16 16' aria-hidden='true' focusable='false'>
-              <path className='iui-check' d='M6,14L0,8l2-2l4,4l8-8l2,2L6,14z' />
-              <path className='iui-check-partial' d='m1 6.5h14v3h-14z' />
-            </svg>
-          )}
+          {!isLoading &&
+            (variant === 'default' ? (
```

Comment:
Maybe move checkboxes html to variables because it is hard to distinct each of them.